kyuubi-zookeeper/src/main/scala/org/apache/kyuubi/zookeeper/EmbeddedZookeeper.scala

```diff
@@ -88,4 +97,15 @@ class EmbeddedZookeeper extends AbstractService("EmbeddedZookeeper") {
     assert(zks != null, s"$getName is in $getServiceState")
     s"$host:${serverFactory.getLocalPort}"
   }
+
+  def resolveRelativePathFromConf(conf: KyuubiConf, configEntry: ConfigEntry[String]): String = {
+    val dirFromConfig = conf.get(configEntry)
+    if (Paths.get(dirFromConfig).isAbsolute) {
+      dirFromConfig
+    } else {
+      val kyuubiHomePrefix = sys.env.getOrElse(KyuubiConf.KYUUBI_HOME, ".") + File.separator
+      kyuubiHomePrefix + dirFromConfig
+    }
```

Comment:
Could it be simplified with
```suggestion
    Paths.get(sys.env.getOrElse(KyuubiConf.KYUUBI_HOME, ".")).resolve(dirFromConfig)
```
?

Reply:
of course, but i want to keep the method to be used in the UT

Reply:
how about
```
def resolvePath(maybeRelativePath: String): String
```
?

since it's "maybe" a relative path

Reply:
Sounds good.  I will changed the method like this:

`  def resolvePathIfRelative(conf: KyuubiConf, configEntry: ConfigEntry[String]): File = {
    val dirFromConfig = conf.get(configEntry)
    Paths.get(sys.env.getOrElse(KyuubiConf.KYUUBI_HOME, ".")).resolve(dirFromConfig).toFile
  }`